static/js/tippyjs.js

```diff
@@ -245,4 +245,35 @@ export function initialize() {
         },
         appendTo: () => document.body,
     });
+
+    delegate("body", {
+        // set tooltip for stream_sorter_toggle buttons.
+        target: ".stream_sorter_toggle .ind-tab",
+        // Add some additional delay when they open
+        // so that regular users don't have to see
+        // them unless they want to.
+        delay: [300, 20],
+
+        // Adjust their placement to `bottom`.
+        placement: "bottom",
+        onShow(instance) {
+            // Set tooltip content of these buttons on the basis of their tabKey.
+            switch (instance.reference.dataset.tabKey) {
+                case "by-stream-name":
+                    instance.setContent($t({defaultMessage: "Sort by name"}));
+                    break;
+
+                case "by-subscriber-count":
+                    instance.setContent($t({defaultMessage: "Sort by number of subscribers"}));
+                    break;
+
+                case "by-weekly-traffic":
+                    instance.setContent($t({defaultMessage: "Sort by estimated weekly traffic"}));
+                    break;
+            }
+        },
```

Comment:
I don't think we need to move this logic here, and in fact it's counterproductive to do so, since it belongs with the string itself. Can you move this part back?

Reply:
Fixed.